### DISCUSSION TITLE
a problem about Convex Optimization I do not know how to do it

### DISCUSSION THREAD

**Discussion by wxy425** [community/community#72815] [Programming Help]
Consider the composite optimization problem
min f (x) + g(x)
x

where f (x) is differentiable and g(x) is a function whose proximal operator is easily available. Let g(x) be a
proper and close function, and xeg g(x) > - 构. The proximal operator of g(x) is defined as proxg(x) = arg n g(u) +  Ⅱu - x Ⅱ2.
1 Proximal Gradient Method for Composite Programming
The proximal gradient method to solve (1) is performed as
x k+1 = prox“kg(xk - “ k ▽f (x k )),
where “ k is a chosen step-size. Your tasks are listed as follows:
1.   Consider the LASSO problem
min  1  ⅡAx - bⅡ + λⅡx Ⅱ1.
Perform numerical experiments on the binary classification problems of the LIBSVM dataset. See https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/
2. Consider the logistic regression problem
min  1   log(1 + exp(-biaiTx)) + λⅡx Ⅱ
i=1
where  Ⅱ .Ⅱ can be either  Ⅱ .Ⅱ1or Ⅱ.Ⅱ2. Perform numerical experiments on the binary classification problem of the LIBSVM dataset. See
https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/

**Comment by queenofcorgis** [comment]
Hi @wxy425 ,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and closed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Asking for homework or test help will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
